.github/workflows/test.yml

```
@@ -0,0 +1,55 @@
+name: Tests
```

Nit for consistency with other Sass repos. (Also, consider renaming file to ci.yml?)

```suggestion
name: CI
```